src/js/components/FrameworkConfiguration.js

```diff
@@ -235,7 +256,7 @@ class FrameworkConfiguration extends Component {
       <div className="flex-item-grow-1">
         <div className="container container-wide">
           {errorsAlert}
-          {preinstall}
+          {warningMessage}
```

Comment:
why did you decide to change this? is it really a "warning"? wouldn't it be possible to split them into two? 
```
{preinstall}
{tandc}
```

Reply:
It must be as one because the content is part of the same paragraph (the preinstall and t&c are concat)